tests/convert_ref_resolution_test.py

```
@@ -0,0 +1,50 @@
+    # Add it to the list of entries that need resolution
+    converter._kp_ref_entries = [mock_entry]
+
+    # Mock _get_referenced_entry and _find_referenced_value to avoid errors when resolving
+    converter._get_referenced_entry = MagicMock(return_value=(None, None, {"login": {"username": "user", "password": "pwd"}}, []))
+    converter._find_referenced_value = MagicMock(return_value="resolved_value")
+    converter._add_bw_entry_to_entries_dict = MagicMock()
+
+    try:
+        converter._resolve_entries_with_references()
+    except TypeError as e:
+        raise AssertionError(f"Failed to resolve entries with None fields: {e}") from e
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)